android/src/main/java/com/reactnativestripesdk/PaymentMethodCreateParamsFactory.kt

```diff
@@ -145,21 +145,22 @@ class PaymentMethodCreateParamsFactory(
 
   @Throws(PaymentMethodCreateParamsException::class)
   private fun createSepaPaymentSetupParams(): ConfirmSetupIntentParams {
-    val billingDetails = billingDetailsParams?.let { it } ?: run {
-      throw PaymentMethodCreateParamsException("You must provide billing details")
-    }
-    val iban = getValOr(params, "iban", null)?.let { it } ?: run {
-      throw PaymentMethodCreateParamsException("You must provide IBAN")
-    }
+    billingDetailsParams?.let {
+      val iban = getValOr(params, "iban", null) ?: run {
+        throw PaymentMethodCreateParamsException("You must provide IBAN")
```

Comment:
In the future we should consider how we localize these error strings for different languages.

Reply:
I would have believed these are for developers, but it looks like the iOS equivalent messages are localized -> https://github.com/stripe/stripe-react-native/blob/master/ios/PaymentMethodFactory.swift#L338-L363

Either way, the server returns a usefully formatted and clear error message in these cases, so we'll probably remove a lot of these custom error messages- https://github.com/stripe/stripe-react-native/pull/821#discussion_r819045389

Reply:
sounds good